src/lambda/commands/invokeLambda.ts

```diff
@@ -80,12 +82,12 @@ export async function invokeLambda(params: {
 
             const inputs: SampleRequest[] = []
 
-            xml2js.parseString(sampleInput, { explicitArray: false }, (err: Error, result: SampleRequestManifest) => {
+            xml2js.parseString(sampleInput, { explicitArray: false }, (err: Error, request: SampleRequestManifest) => {
                 if (err) {
                     return
                 }
 
-                _.forEach(result.requests.request, r => {
+                _.forEach(request.requests.request, r => {
```

Comment:
`(err, result)` is a common JS naming scheme. I'd recommend keeping it that way. It also removes the goofiness of `request.requests.request`.

In that case, change the import statement to something like `result as telemetryResult`

Reply:
I think as a whole, we may want to rename `result` to `telemetryResult`. `result` is too common of a variable name to be constrained to telemetry only.

Reply:
I renamed it to sampleRequest. `result` is allowed in files without telemetry which is most of them. I do not think it is problem to use the name result, which is not that descriptive anyway.

Reply:
It's now back to result :)